scripts/ci/rust_checks.py

```diff
@@ -253,14 +258,17 @@ def docs_slow(results: list[Result]) -> None:
 
 def tests(results: list[Result]) -> None:
     # We first use `--no-run` to measure the time of compiling vs actually running
+    results.append(run_cargo("test", "--all-targets --all-features --no-run"))
+    results.append(run_cargo("nextest", "run --all-targets --all-features"))
```

Comment:
We should ensure it is installed here or in pixi.toml and/or mention nextest in CONTRIBUTING.md

Reply:
well, you wrote "and/or" so I'll let the note in the contributing.md be sufficient for now